docs/multi-index-search.md

```diff
@@ -100,7 +100,7 @@ The 2 stores will in turn fetch new results from Algolia and display them.
 
 **Tip: You can bind any query parameter value to the index by binding an object to `queryParameters`.**
 
-Here is the same example as above with a different syntax, binding the `query`.
+Here is the same example as above with a different syntax, binding the `query`. @TODO: I don't get why we need this example. For every example we need to say why/when you would want to use this.
```

Comment:
This illustrates the use case where you have multiple search experiences on the same page.
Maybe it is obvious that it can work, and we can indeed drop the example entirely.